src/bijectors/corr.jl

```diff
@@ -0,0 +1,109 @@
+# See stan doc for parametrization method:
+# https://mc-stan.org/docs/2_23/reference-manual/correlation-matrix-transform-section.html
+# (7/30/2020) their "manageable expression" is wrong...
+
+struct CorrBijector <: Bijector{2} end
+
+(b::CorrBijector)(X::AbstractMatrix{<:Real}) = link_lkj(X)
+(b::CorrBijector)(X::AbstractArray{<:AbstractMatrix{<:Real}}) = map(b, X)
+
+(ib::Inverse{<:CorrBijector})(Y::AbstractMatrix{<:Real}) = inv_link_lkj(Y)
+(ib::Inverse{<:CorrBijector})(Y::AbstractArray{<:AbstractMatrix{<:Real}}) = map(ib, Y)
+
+
+logabsdetjac(::Inverse{CorrBijector}, y::AbstractMatrix{<:Real}) = log_abs_det_jac_lkj(y)
+function logabsdetjac(b::CorrBijector, X::AbstractMatrix{<:Real})
+    
+    return -log_abs_det_jac_lkj(b(X)) # It may be more efficient if we can use un-contraint value to prevent call of b
+end
+logabsdetjac(b::CorrBijector, X::AbstractArray{<:AbstractMatrix{<:Real}}) = mapvcat(X) do x
+    logabsdetjac(b, x)
+end
+
+
+function log_abs_det_jac_lkj(y)
+    # it's defined on inverse mapping
+    K = size(y, 1)
+    
+    z = tanh.(y)
+    left = eltype(y)(0)
+    for i = 1:(K-1), j = (i+1):K
```

Comment:
Julia uses column-major order, and hence it is more efficient to loop over rows in each column than to loop over columns for each row. I.e., the loop over index `i` should be the inner loop, and the one over `j` the outer loop.

Reply:
Should we also make all these loops `@inbounds`? Possibly together with adding a `@assert size(y, 1) == size(y, 2)` statement above or something?

Reply:
Changed looping order and added `@inbounds`.